tests/decorators/decorators.go

```diff
@@ -0,0 +1,25 @@
+package decorators
+
+import . "github.com/onsi/ginkgo/v2"
+
+var (
```

Comment:
As a follow-up, we should analyze and design what labels we need.

Reply:
Indeed! The idea was just that!

1. Start using labels
2. Analyze what labels we need
3. Move to the new labels

Thanks

Reply:
Great! Do we have a tracker issue?